### ISSUE TITLE
No instance nor default method for class operation mappend

### ISSUE THREAD

**Issue by jknaus** [matterhorn-chat/matterhorn#744]
I just tried out matterhorn for the first time.
It seems to connect fine but it exits directly after login with this message:
```
matterhorn: src/Matterhorn/Types/Channels.hs:342:10-45: No instance nor default method for class operation mappend
```
I am on Fedora34. Is this a dependency missing problem?

**Comment by jtdaugherty** [comment]
HI @jknaus - thank you for reporting this! This is not a problem with your system. It is a problem with the build. (Although to be explicit, our releases are officially supporting Fedora 29, not Fedora 34, but if our releases work on Fedora 34, that's great.)

We have a fix in the works but it'll be a week or so before we can get a new release out to you. Sorry for the delay!

**Comment by jtdaugherty** [comment]
For our records, the fix is:
```haskell
instance Monoid (AllMyChannels ClientChannel) where
  mempty = noChannels
#if !MIN_VERSION_base(4,11,0)
  mappend = (<>)
#endif
```

**Comment by jtdaugherty** [comment]
This issue is now fixed by the latest release, [50200.14.1](https://github.com/matterhorn-chat/matterhorn/releases/tag/50200.14.1). Thanks again!
